python/src/aiconfig/editor/server/server.py

```diff
@@ -119,12 +119,23 @@ def create() -> FlaskResponse:
 async def run() -> FlaskResponse:
     state = get_server_state(app)
     request_json = request.get_json()
-    prompt_name = request_json.get("prompt_name", None)
-    stream = request_json.get("stream", True)
+    prompt_name : str = request_json.get("prompt_name", None)
+    if prompt_name is None:
+        return HttpResponseWithAIConfig(
+            message="No prompt name provided, cannot execute `run` command",
+            code=400,
+            aiconfig=None,
+        ).to_flask_format()
+    params : str = request_json.get("params", None)
+    stream : bool = request_json.get("stream", True)
```

Comment:
nits: 
prompt_name is a str | None
please use the linter and/or vscode for pyright + black formatting.

# autofix just these files, one-liner:
`fd --glob '*.py'  python/src/aiconfig/editor/server | xargs python -m 'scripts.lint' --mode=fix --files`

Reply:
Awesome thanks for the auto-fix command!